### PR TITLE
ghci: add page

### PR DESCRIPTION
**Pull request by mira01**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/ghci.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# ghci
+
+> Interactive Haskell environment.
+
+- Run environment and load a module:
+
+`ghci {{module}}`
+
+- Use current directory for command history file:
```
Comment (sbrl) [inline pages/common/ghci.md]: Perhaps `Use the current directory for the command`..... would read  better?

File: pages/common/ghci.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# ghci
+
+> Interactive Haskell environment.
```
Comment (agnivade) [inline pages/common/ghci.md]: We should add the words "Glasgow Haskell compiler" ..

File: pages/common/ghci.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+- Run environment and load a module:
```
Comment (agnivade) [inline pages/common/ghci.md]: Also please add the simplest example of `ghci` and describe it.

File: pages/common/ghci.md (hunk 2 above)
Comment (sbrl) [reply pages/common/ghci.md]: Perhaps `Interactive Glasgow Haskell environment.` would work?

Comment (sbrl) [reply pages/common/ghci.md]: Ah! I see. `Interactive Glasgow Haskell Compiler environment.` then?

**Pull request closed without merging**